baseplate/lib/metrics.py

```diff
@@ -563,8 +563,12 @@ def metrics_client_from_config(raw_config: config.RawConfig) -> Client:
         raw_config,
         {
             "metrics": {
-                "namespace": config.Optional(config.String, default=""),
-                "endpoint": config.Optional(config.Endpoint),
+                "namespace": config.OptionalDefaultFromEnv(
```

Comment:
Note that `namespace` is soft-deprecated in favor of configuring this in telegraf. In fact, when doing tagged metrics `namespace` is disallowed.

Reply:
I'll remove this one then :)